### PR TITLE
391 release 4.2.2

### PR DESCRIPTION
**Pull request by vcerenu**
Release Issue https://github.com/wazuh/wazuh-puppet/issues/391
This PR adds the latest changes to branch 4.2.2

### CODE REVIEW

**Review by teddytpc1 — APPROVED** [review]
LGTM

**Comment by lastranikos** [comment]
LGTM

**Review by lastranikos — APPROVED** [review]
LGTM

File: manifests/params_agent.pp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Wazuh App Copyright (C) 2021 Wazuh Inc. (License GPLv2)
 # Wazuh-Agent configuration parameters
 class wazuh::params_agent {
-  $agent_package_version = '4.2.1-1'
+  $agent_package_version = '4.2.2'
```
Comment (okynos) [inline manifests/params_agent.pp:4]: Add `-1`

File: manifests/params_manager.pp [hunk 2]
```diff
@@ -5,7 +5,7 @@
     'Linux': {
 
     # Installation
-      $server_package_version                          = '4.2.1-1'
+      $server_package_version                          = '4.2.2'
```
Comment (okynos) [inline manifests/params_manager.pp:8]: Same here.

**Review by okynos — CHANGES_REQUESTED** [review]
Review please, GJ.

File: manifests/params_agent.pp (hunk 1 above)
Comment (vcerenu) [reply manifests/params_agent.pp:4]: This change was made in commit https://github.com/wazuh/wazuh-puppet/commit/bfab4c0c1bf761306dc9882808ddcfb44d3d846d

File: manifests/params_manager.pp (hunk 2 above)
Comment (vcerenu) [reply manifests/params_manager.pp:8]: This change was made in commit https://github.com/wazuh/wazuh-puppet/commit/bfab4c0c1bf761306dc9882808ddcfb44d3d846d

Comment (vcerenu) [reply manifests/params_manager.pp]: Fixed in commit https://github.com/wazuh/wazuh-puppet/pull/393/commits/7342c3713d0d64202e9a9dc49c53f4bf4887505f

**Review by alberpilot — APPROVED** [review]
LGTM

**Review by teddytpc1 — APPROVED** [review]
LGTM

**Review by okynos — APPROVED** [review]
LGTM.

**Pull request merged**